Sources/NIOHTTP1/HTTPEncoder.swift

```diff
@@ -190,3 +182,121 @@ public final class HTTPResponseEncoder: ChannelOutboundHandler {
         }
     }
 }
+
+private extension ByteBuffer {
+    private mutating func write(status: HTTPResponseStatus) {
+        if case .ok = status {
+            // Optimize for 200 ok, which should be the most likely code (...hopefully).
+            self.write(staticString: status200)
+        } else {
+            self.write(string: String(status.code))
+            self.write(string: " ")
```

Comment:
This should be `staticString`.

Reply:
@Lukasa we should really probably make this `self.write(bytes: [32])` or `self.write(integer: 32, as: UInt8.self)`. No idea what's fastest.

Reply:
@Lukasa for one character I'm also not sure if `staticString` is actually faster than `string`. No idea